frontend/packages/console-app/console-extensions.json

```diff
@@ -628,6 +628,9 @@
         "falseValue": false,
         "defaultValue": false
       }
+    },
+    "flags": {
+      "required": ["PROMETHEUS", "MONITORING", "CAN_GET_NS"]
     }
```

Comment:
Tested it. It is working fine but I have one question, why PROMETHEUS and MONITORING flags are added?

Reply:
Normally it's observed that normal users do not have access to the cluster monitoring resources (Observe tab charts). Therefore, this can also be a good condition to check whether the user is an admin or a normal user.

That may be the reason these were added.
It's mostly safe as Kubeadmin always has access to cluster-monitoring.